src/components/MessagesList/MessagesList.vue

```diff
@@ -1157,10 +1157,12 @@ export default {
 
 			// TODO: doesn't work if chat is hidden. Need to store
 			// delayed 'shouldScroll' and call after chat is visible
-			scrollElement.scrollIntoView({
-				behavior: smooth ? 'smooth' : 'auto',
-				block: 'center',
-				inline: 'nearest',
+			this.$nextTick(() => {
```

Comment:
It's actually working, but it's followed by `scrollToBottom()`:

https://github.com/nextcloud/spreed/blob/e1a4ae8faf89965da34c149983a5b1ece8a4d14d/src/components/MessagesList/MessagesList.vue#L301